cmd/frontend/db/orgs.go

```diff
@@ -112,20 +111,17 @@ func (o *orgs) List(ctx context.Context, opt *OrgsListOptions) ([]*types.Org, er
 	if opt == nil {
 		opt = &OrgsListOptions{}
 	}
-	conds := o.listSQL(*opt)
-
-	q := sqlf.Sprintf("WHERE %s ORDER BY id ASC %s", sqlf.Join(conds, "AND"), opt.LimitOffset.SQL())
+	q := sqlf.Sprintf("WHERE %s ORDER BY id ASC %s", o.listSQL(*opt), opt.LimitOffset.SQL())
 	return o.getBySQL(ctx, q.Query(sqlf.PostgresBindVar), q.Args()...)
 }
 
-func (*orgs) listSQL(opt OrgsListOptions) (conds []*sqlf.Query) {
-	conds = []*sqlf.Query{sqlf.Sprintf("TRUE")}
-	conds = append(conds, sqlf.Sprintf("deleted_at IS NULL"))
+func (*orgs) listSQL(opt OrgsListOptions) *sqlf.Query {
+	conds := []*sqlf.Query{sqlf.Sprintf("deleted_at IS NULL")}
 	if opt.Query != "" {
 		query := "%" + opt.Query + "%"
 		conds = append(conds, sqlf.Sprintf("name ILIKE %s OR display_name ILIKE %s", query, query))
```

Comment:
alternative: I'm not sure what is better, but I thought this might be since its easier to read. But your solution is more future proof. More than likely stick with your current solution :)

```suggestion
		conds = append(conds, sqlf.Sprintf("(name ILIKE %s OR display_name ILIKE %s)", query, query))
```